lib/Ast.ml

```diff
@@ -131,6 +131,7 @@ type 'e sizedtype =
   | SArray of 'e sizedtype * 'e
 [@@deriving sexp, compare, map, hash]
 
+(** remove_size [st] converts st from a sizedtype to an unsizedtype. *)
```

Comment:
I'd prefer to document the return.  And I don't think we need to repeat the name of the function.
```
(** Return the unsized type corresponding to the argument type. *)
```
It's a bit confusing here because there's no real argument `st` anywhere to document.

Reply:
For some reason, the Jane Street style guide repeats the name of the function: https://opensource.janestreet.com/standards/ (under Comments). It doesn't seem to show up when they generate the ocamldoc webpages at all, which is weird. It shows up like this in my editor: 
![image](https://user-images.githubusercontent.com/805356/52544722-84e25500-2d80-11e9-83b9-7d80aec1c3a1.png)

Do you think we should deviate from their style guide? I was kind of going with it for now just because it's nice not to have to choose things.

Reply:
I think that repetition is related to your other comment in that you might want to e.g. show that a point-free function definition actually takes a specific arg, or something?